plugins/ipam/dhcp/README.md

```diff
@@ -18,7 +18,7 @@ $ ./dhcp daemon
 
 If given `-pidfile <path>` arguments after 'daemon', the dhcp plugin will write
 its PID to the given file.
-If given `-hostprefix <prefix>` arguments after 'daemon', the dhcp plugin will use this prefix for netns as `<prefix>/<original netns>`. It could be used in case of running dhcp daemon as container.
+If given `-hostprefix <prefix>` arguments after 'daemon', the dhcp plugin will use this prefix for DHCP socket as `<prefix>/run/cni/dhcp.sock`. It could be used in case of running dhcp daemon as container.
```

Comment:
From my reading of the code the parameter is now used for _both_ netns and socket.
The context is that the daemon wants to send DHCP messages from the interface of other containers, so it wants to reach into their namespace.
Maybe you could say something like "use this prefix for references to the host filesystem, e.g. to access netns and the unix socket" ?

Reply:
Changed. Thank you for your valuable comment!